catalog/transformer/catalog-transformer-mgmp/src/main/java/org/codice/alliance/catalog/transformer/mgmp/MgmpTransformer.java

```diff
@@ -0,0 +1,604 @@
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_PROJECTION_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_PROJECTION_NUMBER_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_GRID_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_GRID_NUMBER_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_GRID_WKT_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_PROJECTION_WKT_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_ELLIPSOID_WKT_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_DATUM_WKT_TYPE_PATH,
+
+                MgmpConstants.ISR_COVERAGE_CATEGORY_PATH,
+                MgmpConstants.ISR_COVERAGE_COMMENT_PATH,
+                MgmpConstants.ISR_IMAGE_COMMENT_PATH,
+                MgmpConstants.ISR_IMAGE_DESCRIPTION_PATH,
+                MgmpConstants.ISR_MD_IMAGE_COMMENT_PATH,
+                MgmpConstants.ISR_MD_IMAGE_DESCRIPTION_PATH,
+                MgmpConstants.ISR_VIDEO_COMMENT_PATH,
+                MgmpConstants.ISR_VIDEO_DESCRIPTION_PATH,
+
+                MgmpConstants.METADATA_ORIGINATOR_SECURITY_PATH,
+                MgmpConstants.METADATA_RELEASABILITY_PATH,
+                MgmpConstants.RESOURCE_SECURITY_RELEASABILITY_PATH,
+                MgmpConstants.NIIRS_PATH,
+                MgmpConstants.NIIRS_RATING_PATH,
+                MgmpConstants.RESOURCE_SECURITY_PATH,
+                MgmpConstants.METADATA_SECURITY_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_CRS_WKT_TYPE_PATH)
+                .stream()
+                .map(this::toPath)
+                .forEach(paths::add);
+
+        paths.addAll(super.buildPaths());
+        return paths;
+    }
+
+    @Override
+    public MetacardImpl toMetacard(final XstreamPathValueTracker pathValueTracker, String id) {
+        MetacardImpl metacard = new MetacardImpl(mgmpMetacardType);
+        super.setMetacardAttributes(metacard, id, pathValueTracker);
+        addMetacardIsrInformation(pathValueTracker, metacard);
+        addMetacardNiirsInformation(pathValueTracker, metacard);
+        addMetacardLanguage(pathValueTracker, metacard);
+        addMetacardCloudCoverPercentage(pathValueTracker, metacard);
+        addMetacardResourceSecurityClassification(pathValueTracker, metacard);
+        addMetacardResourceSecurityReleasability(pathValueTracker, metacard);
+        addMetacardResourceSecurityOriginatorClassification(pathValueTracker, metacard);
+        addMetacardMetadataSecurityReleasability(pathValueTracker, metacard);
+        addMetacardMetadataSecurityOriginatorClassification(pathValueTracker, metacard);
+        addMetacardMetadataSecurityClassification(pathValueTracker, metacard);
+        addMetacardDataQuality(metacard);
+        metacard.setAttribute(Media.TYPE, MgmpConstants.DEFAULT_MEDIA_TYPE);
```

Comment:
this is supposed to be the media type of the resource not the metadata.  Not sure we can get that from the MGMP xml